### PR TITLE
rustup "Cheaper doc comments"

### PR DESCRIPTION
**Pull request by lzutao**
rustup https://github.com/rust-lang/rust/pull/65750
changelog: none

### CODE REVIEW

File: clippy_lints/src/utils/attrs.rs [hunk 1]
```diff
@@ -57,46 +57,50 @@ pub fn get_attr<'a>(
...
-                            Applicability::MachineApplicable,
-                        );
-                        db.emit();
-                        false
-                    },
-                    DeprecationStatus::None => {
-                        db.cancel();
-                        attr_segments[1].ident.to_string() == name
-                    },
+        if attr.is_doc_comment() {
```
Comment (lzutao) [inline clippy_lints/src/utils/attrs.rs]: I think i should split this closure to a separate function.

**Comment by flip1995** [comment]
closing in favor of #4788 (smaller diff)

**Comment by mati865** [comment]
https://github.com/rust-lang/rust-clippy/pull/4787 had even smaller diff than #4788

**Comment by lzutao** [comment]
I think #4788 is nicer.

**Pull request closed without merging**